packages/main/src/TimePickerPopover.hbs

```diff
@@ -36,7 +36,6 @@
 				._items="{{secondsArray}}"
 				@click="{{handleSliderClicked}}"
 				class="ui5-timepicker-wheelslider ui5-timepicker-seconds-wheelslider"
-				cyclic
```

Comment:
Is this accidental? Due do this change probably, the seconds slider can run out of items if you scroll long.